src/api/spec/controllers/webui/project_controller_spec.rb

```diff
@@ -799,4 +799,129 @@
       it { expect(user.home_project.repositories.find_by(name: 'valid_name').repository_architectures.count).to eq(1) }
     end
   end
+
+  describe 'POST #save_distributions' do
```

Comment:
@hennevogel  Talked to @ChrisBr at IRC about it and we agreed on merging this... after he rebase and add his own tests he can get rid of this... sorry!